src/main/java/com/shootdoori/match/entity/MercenaryReview.java

```diff
@@ -51,27 +47,19 @@ public class MercenaryReview {
     @JoinColumn(name = "mercenary_user_id", nullable = false)
     private User mercenaryUser;
 
-    @Column(name = "mercenary_name", nullable = false, length = 100)
-    private String mercenaryName;
+    @Min(1)
+    @Max(5)
+    @Column(name = "rating", nullable = false)
+    private Integer rating;
 
-    @Min(1) @Max(5)
-    @Column(name = "overall_rating", nullable = false)
-    private int overallRating;
+    @Column(name = "punctuality_review")
+    private ReviewBinaryEvaluation punctualityReview;
 
-    @Min(1) @Max(5)
-    @Column(name = "skill_rating", nullable = false)
-    private int skillRating;
+    @Column(name = "sportsmanship_review")
+    private ReviewBinaryEvaluation sportsmanshipReview;
 
-    @Min(1) @Max(5)
-    @Column(name = "manner_rating", nullable = false)
-    private int mannerRating;
-
-    @Min(1) @Max(5)
-    @Column(name = "communication_rating", nullable = false)
-    private int communicationRating;
-
-    @Column(name = "comment", length = 300)
-    private String comment;
+    @Column(name = "skill_level_review")
+    private ReviewSkillLevel skillLevelReview;
```

Comment:
```suggestion
    @Enumerated(EnumType.STRING)
    @Column(name = "skill_level_review")
    private ReviewSkillLevel skillLevelReview;
```

`enum`타입을 entity에 그대로 매핑이 안될텐데, 어떻게 생각하시나요? 🤔